### PR TITLE
Add displayName for SAP HANA template parameters

### PR DESCRIPTION
**Pull request by yaacov**
**What this PR does / why we need it**:
The UI will let users fill in the required parameters for SAP HANA template
https://issues.redhat.com/browse/CNV-14867
The UI will use the `displayName` field to correctly display titles for the parameter inputs

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/CNV-14867

**Special notes for your reviewer**:
UI pull request to use this fields: https://github.com/openshift/console/pull/10617

**Release note**:

```release-note
Add displayName field to correctly display titles for the parameter inputs for SAP HANA template in the UI
```

### CODE REVIEW

**Comment by yaacov** [comment]
@rmohr @dominikholler @avivtur @tnisan please review

File: templates/rhel8.saphana.tpl.yaml [hunk 1]
```diff
@@ -153,42 +153,55 @@ objects:
...
   - name: SRC_CONTAINERDISK
     description: Name of the source container disk to import
+    displayName: Source container disk
     value: "docker://registry.access.redhat.com/rhel8/rhel-guest-image:8.4.0"
   - description: Randomized password for the cloud-init user {{ cloudusername }}
+    displayName: Cloud user password
     from: '[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}'
     generate: expression
     name: CLOUD_USER_PASSWORD
   - description: Name of the node where this VM needs to run
```
Comment (yaacov) [inline templates/rhel8.saphana.tpl.yaml:204]: This is a required field, but the UI will show it last, after the non required fields, do we want to raise it up ?

Comment (rmohr) [reply templates/rhel8.saphana.tpl.yaml:204]: @yaacov sounds good to me :)

File: templates/rhel8.saphana.tpl.yaml [hunk 2]
```diff
@@ -153,42 +153,55 @@ objects:
...
   - description: Amount of sockets
+    displayName: CPU sockets
     name: CPU_SOCKETS
     value: "1"
   - description: Name of the sriov network1
+    displayName: sriov network1
     name: SRIOV_NETWORK_NAME1
     value: "default/sriov-net1"
   - description: Name of the sriov network2
+    displayName: sriov network2
```
Comment (avivtur) [inline templates/rhel8.saphana.tpl.yaml:184]: same

File: templates/rhel8.saphana.tpl.yaml [hunk 3]
```diff
@@ -153,42 +153,55 @@ objects:
...
     name: SRIOV_NETWORK_NAME2
     value: "default/sriov-net2"
   - description: Name of the sriov network3
+    displayName: sriov network3
```
Comment (avivtur) [inline templates/rhel8.saphana.tpl.yaml:188]: same

File: templates/rhel8.saphana.tpl.yaml [hunk 4]
```diff
@@ -153,42 +153,55 @@ objects:
...
   - description: Amount of threads
+    displayName: CPU threads
     name: CPU_THREADS
     value: "1"
   - description: Amount of sockets
+    displayName: CPU sockets
     name: CPU_SOCKETS
     value: "1"
   - description: Name of the sriov network1
+    displayName: sriov network1
```
Comment (avivtur) [inline templates/rhel8.saphana.tpl.yaml:180]: nitpick suggestion: when adding a network interface on the web UI modal,
the naming is `SR-IOV` and not `sriov` maybe it's best to change for consistancy

File: templates/rhel8.saphana.tpl.yaml [hunk 5]
```diff
@@ -153,42 +153,55 @@ objects:
...
   - description: Amount of memory
+    displayName: Memory
     name: MEMORY
     value: 24Gi
   - description: Amount of memory overhead for qemu
+    displayName: Memory overhead
     name: MEMORY_OVERHEAD
     value: 44Gi
   - description: Amount of cores
+    displayName: CPU Cores
```
Comment (avivtur) [inline templates/rhel8.saphana.tpl.yaml:168]: nitpick: please change `Cores` to `cores` for consistency

**Review by avivtur — COMMENTED** [review]
just some minor suggestions

**Comment by yaacov** [comment]
Changes from last review iteration:
- Capitalize titles for CPU and Memory
CPU cores -> CPU Cores
CPU threads -> CPU Trheads
Memory overhead -> Memory Overhead

- Capitalize acronym sriov
sriov -> SR-IOV

-  Move required field node name to be before none required fields

@lmandavi can you look at the texts added in this pull request ?
@avivtur @ksimon1 please re-review

File: templates/rhel8.saphana.tpl.yaml (hunk 5 above)
Comment (yaacov) [reply templates/rhel8.saphana.tpl.yaml:172]: @lmandavi hi, do we need to capitalize here ?

**Comment by yaacov** [comment]
> /approve cancel

@akrejcir hi, any change I need to do ?

**Pull request merged**